src/Workspaces/Core/Portable/FlowAnalysis/SymbolUsageAnalysis/SymbolUsageAnalysis.DataFlowAnalyzer.FlowGraphAnalysisData.cs

```diff
@@ -0,0 +1,499 @@
+                        _lValueFlowCapturesMap.Add(captureId, captures);
+                    }
+
+                    captures.Add((symbol, operation));
+                }
+
+                public override void OnLValueDereferenceFound(CaptureId captureId)
+                {
+                    if (_lValueFlowCapturesMap.TryGetValue(captureId, out var captures))
+                    {
+                        var mayBeWritten = captures.Count > 1;
+                        foreach (var (symbol, write) in captures)
+                        {
+                            OnWriteReferenceFound(symbol, write, mayBeWritten);
+                        }
+                    }
+                }
+
+                protected override BasicBlockAnalysisData AnalyzeLocalFunctionInvocationCore(IMethodSymbol localFunction, CancellationToken cancellationToken)
+                {
+                    Debug.Assert(localFunction.IsLocalFunction());
+
+                    cancellationToken.ThrowIfCancellationRequested();
+                    if (!_localFunctionTargetsToAccessingCfgMap.TryGetValue(localFunction, out var accessingCfg))
+                    {
+                        accessingCfg = ControlFlowGraph;
+                    }
+
+                    var localFunctionCfg = accessingCfg.GetLocalFunctionControlFlowGraphInScope(localFunction, cancellationToken);
+                    return _analyzeLocalFunctionOrLambdaInvocation(localFunction, localFunctionCfg, this, cancellationToken);
+                }
+
+                protected override BasicBlockAnalysisData AnalyzeLambdaInvocationCore(IFlowAnonymousFunctionOperation lambda, CancellationToken cancellationToken)
+                {
+                    cancellationToken.ThrowIfCancellationRequested();
+                    if (!_lambdaTargetsToAccessingCfgMap.TryGetValue(lambda, out var accessingCfg))
+                    {
+                        accessingCfg = ControlFlowGraph;
+                    }
+
+                    var lambdaCfg = accessingCfg.GetAnonymousFunctionControlFlowGraphInScope(lambda, cancellationToken);
+                    return _analyzeLocalFunctionOrLambdaInvocation(lambda.Symbol, lambdaCfg, this, cancellationToken);
+                }
+
+                public override bool IsTrackingDelegateCreationTargets => true;
+                public override void SetTargetsFromSymbolForDelegate(IOperation write, ISymbol symbol)
+                {
+                    // Transfer reaching delegate creation targets when assigning from a local/parameter symbol
+                    // that has known set of potential delegate creation targets. For example, this method will be called
+                    // for definition 'y' from symbol 'x' for below code:
+                    //      Action x = () => { };
+                    //      Action y = x;
+
+                    var targetsBuilder = PooledHashSet<IOperation>.GetInstance();
```

Comment:
nit: remove the empty line

Reply:
and similar below in the file

---
In reply to: [234061368](https://github.com/dotnet/roslyn/pull/31025#discussion_r234061368) [](ancestors = 234061368)